content/en/docs/concepts/storage/volumes.md

```diff
@@ -11,221 +11,289 @@ weight: 10
+other volumes. Each container in the pod's configuration must independently specify where to
+mount each volume. 
 
 ```yaml
 apiVersion: v1
 kind: Pod
 metadata:
-  name: test-ebs
+  name: sample-cache-pod
 spec:
-  containers:
-  - image: k8s.gcr.io/test-webserver
-    name: test-container
-    volumeMounts:
-    - mountPath: /test-ebs
-      name: test-volume
   volumes:
-  - name: test-volume
-    # This AWS EBS volume must already exist.
-    awsElasticBlockStore:
-      volumeID: "<volume id>"
-      fsType: ext4
+    [...]
+  containers:
+    - name: test
+      image: busybox
+      volumeMounts:
+        - name: cache-volume
+          mountPath: /etc/cache
+
 ```
 
-#### AWS EBS CSI migration
+### Core Volume Types
 
-{{< feature-state for_k8s_version="v1.17" state="beta" >}}
+Kubernetes supports several types of `Volumes`. These core volume types are 
+integrated into Kubernetes, and include network storage volume types and storage provided by the pod itself. 
 
-The `CSIMigration` feature for `awsElasticBlockStore`, when enabled, redirects
-all plugin operations from the existing in-tree plugin to the `ebs.csi.aws.com` Container
-Storage Interface (CSI) driver. In order to use this feature, the [AWS EBS CSI
-driver](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
-must be installed on the cluster and the `CSIMigration` and `CSIMigrationAWS`
-beta features must be enabled.
+Core Volume Types:
+   * [cephfs](#cephfs)
+   * [configMap](#configmap)
+   * [downwardAPI](#downwardapi)
+   * [emptyDir](#emptydir)
+   * [fc (fibre channel)](#fc)
+   * [gitRepo (deprecated)](#gitrepo)
+   * [glusterfs](#glusterfs)
+   * [hostPath](#hostpath)
+   * [iscsi](#iscsi)
+   * [local](#local)
+   * [nfs](#nfs)
+   * [projected](#projected)
+   * [rbd](#rbd)
+   * [secret](#secret)
 
-#### AWS EBS CSI migration complete
 
-{{< feature-state for_k8s_version="v1.17" state="alpha" >}}
+### Volume Plugin Interfaces
 
-To disable the `awsElasticBlockStore` storage plugin from being loaded by the controller manager
-and the kubelet, set the `CSIMigrationAWSComplete` flag to `true`. This feature requires the `ebs.csi.aws.com` Container Storage Interface (CSI) driver installed on all worker nodes.
+Kubernetes implements the Container Storage Interface (CSI). CSI is the modern way to communicate with out-of-tree plugins for externaly managed volumes. More specifically, CSI is a standard for exposing arbitrary block and file storage systems to containerized workloads. Learn more with the [CSI docs](https://kubernetes-csi.github.io/docs/).
 
-### azureDisk {#azuredisk}
+*Out-of-tree* storage plugins are not part of the kubernetes binary. 
 
-The `azureDisk` volume type mounts a Microsoft Azure [Data Disk](https://docs.microsoft.com/en-us/azure/aks/csi-storage-drivers) into a pod.
+`FlexVolume` is an alternate way to communicate with out of tree volume plugins. `FlexVolume` was developed before CSI.
```

Comment:
```suggestion
[FlexVolume](#flexvolume) is an another mechanism for storage plugins. Unlike CSI, FlexVolume is
more closely tied to Kubernetes and its storage implementation. FlexVolume was developed before CSI.
```
I would not write `FlexVolume`, though, because when readers are specifying it as a volume type they are most likely to write `flexVolume`. To me, using FlexVolume just as a proper noun is OK.